autopts/bot/iut_config/mynewt.py

```diff
@@ -48,14 +48,17 @@
             'BLE_SM_SC': '1',
             'BLE_SM_SC_ONLY': '0',
             'BLE_SM_LVL': '4',
+            'BLE_SM_MITM': '1',
         },
         "test_cases": [
             'GAP/SEC/SEM/BV-21-C',
             'GAP/SEC/SEM/BV-22-C',
             'GAP/SEC/SEM/BV-24-C',
             'GAP/SEC/SEM/BV-26-C',
             'GAP/SEC/SEM/BV-27-C',
-            'GAP/SEC/SEM/BV-45-C'
+            'GAP/SEC/SEM/BV-45-C',
+            'GAP/SEC/SEM/BV-58-C',
+            'GAP/SEC/SEM/BV-61-C'
```

Comment:
lets terminate that with ,  to avoid future issues if one miss to add it when extending test_cases :-)

Reply:
Done